### ISSUE TITLE
exchange field is missing from search stocks by symbol

### ISSUE THREAD

**Issue by rdavis120** [numberjuani/ibkr-cp-api-client#15]
The search stocks by symbol endpoint returns a contracts field with only the fields conid and exchange,  but its being mapped to the type Vec<Contract>.  However Contract doesn't have the exchange field since it comes from another endpoint.  I would propose either adding the ```exchange``` field to ```Contract``` or creating a new struct that has fewer fields specifically for the ```contracts``` field in StockContractInfo.  Please let me know your thoughts.

```rust
pub struct StockContractInfo {
    pub asset_class: AssetClass,
    pub chinese_name: Option<String>,
    pub contracts: Vec<Contract>,
    pub name: String,
}
````

**Comment by numberjuani** [comment]
sure, want to open an PR for it?

**Comment by numberjuani** [comment]
fixed

**Comment by numberjuani** [comment]
fixed in latest commit, no PR, version 0.3
